programs/server/config.xml

```diff
@@ -1133,6 +1133,16 @@
         <ttl>event_date + INTERVAL 30 DAY DELETE</ttl>
         -->
 
+        <!--
+            ORDER BY expr: https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#order_by
+            Example:
+                event_date, event_time
+                event_date, type, query_id
+                event_date, event_time, initial_query_id
+
+        <order_by>event_date, event_time, initial_query_id</order_by>
```

Comment:
What about configuration for `<trace_log>`, `<query_thread_log>` below? Please update them accordingly (if needed).

Reply:
No, it's not ncessary, because it's a little bit stupid to add this configuration to every system log table. The template in the first system log table is sufficient :)